Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "themis",
+    products: [
+        .library(
+            name: "themis",
+            targets: ["themis"]),
+    ],
+    dependencies: [],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
```

Comment:
I guess you could replace this with more informative instructions for maintainers, like in CLOpenSSL repo.